centaur/src/main/resources/standardTestCases/docker_hash/docker_hash_quay.wdl

```diff
@@ -3,7 +3,7 @@ task quay {
         echo "hello"
     }
     runtime {
-        docker: "quay.io/broadinstitute/cromwell-docker-test:centaur"
+        docker: "quay.io/broadinstitute/cromwell-tools:v2.4.1"
```

Comment:
Since we don't really care what the image is, let's use something that's actively updated.
```suggestion
        docker: "quay.io/fedora/fedora-minimal:latest"
```

Reply:
Agreed in principle, but the `.test` file is checking the hash, so we actually DON'T want something that changes. Could do `quay.io/fedora/fedora-minimal:41`?

Reply:
Good point. `41` will still float, but `quay.io/fedora/fedora-minimal@sha256:63f785bf6185a63332fd9ccdaaabc66f47bc63564e2cecc6f99194002ca73ac3` should do the trick.